common/src/main/java/org/opensearch/ml/common/FunctionName.java

```diff
@@ -8,6 +8,9 @@
 import java.util.HashSet;
 import java.util.Set;
 
+// Please strictly added new FunctionName to the last line
+// and avoid altering the order of the existing FunctionName,
+// or it will break the backward compatibility!
```

Comment:
Is it possible to explain why it breaks the backward compatibility for not having the order?

Reply:
This is because java enumerate class is actually use the order to locate the object (You can treat it as a python dictionary with key to be a number and value to be the object you want to retrieve). Therefore if there's an order difference between older version and newer version it will locate to a value we don't want.